### ISSUE TITLE
qlfasm script in release v1.1.0 references invalid path

### ISSUE THREAD

**Issue by cibomahto** [QuickLogic-Corp/quicklogic-fpga-toolchain#16]
In the file $INSTALL_DIR/install/bin/python/qlfasm, the location of python is hardcoded as:

```
#!/home/kkumar/symbiflow-arch-defs/build/env/conda/bin/python3
# EASY-INSTALL-ENTRY-SCRIPT: 'quicklogic-fasm','console_scripts','qlfasm'
```

This prevents some examples from 'symbiflow-examples' from compiling. The path '/home/kkumar' appears in a variety of other files as well.

**Comment by kkumar23** [comment]
@cibomahto This is a known issue and we are planning for a fix in the next release. Meanwhile workaround would be to change the python3 path to the local install.

**Comment by kkumar23** [comment]
Fixed in symbiflow 1.2.0 version
